client/app/fpaas/fingerprint/accordion/accordion.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'accordion',
+  templateUrl: './app/fpaas/fingerprint/accordion/accordion.component.html',
+  styleUrls: ['./app/fpaas/fingerprint/fingerprint.component.css']
+})
+
+export class AccordionComponent {
+
+  accordionState: any = null;
+  tab: string;
+
+  public data: any = [{
+    title: 'Fingerprint Data',
+    rows: {
+           "Code Count": 654,
+           Code: "eJzVmmuOJDcOhK-kFynpOKQe9z_" +
+            "CfuwF3eJzVmmuOJDcOhK-kFynpOKQe9z_CfuwF3"
+         }
+  },
+  {
+    title: 'Metadata',
+    rows: {
+           artist: '',
```

Comment:
### Nitpick

Can we have the attributes indented 2 spaces from the opening brace?